src/core/Akka.API.Tests/CoreAPISpec.ApproveRemote.verified.txt

```diff
@@ -580,6 +580,58 @@ namespace Akka.Remote.Transport
         public FailureInjectorProvider() { }
         public Akka.Remote.Transport.Transport Create(Akka.Remote.Transport.Transport wrappedTransport, Akka.Actor.ExtendedActorSystem system) { }
     }
+    public class FailureInjectorTransportAdapter : Akka.Remote.Transport.AbstractTransportAdapter, Akka.Remote.Transport.IAssociationEventListener
```

Comment:
LGTM - this is sufficiently obscure that it's not going to regularly pop-up in end-user code, but still accessible to an extent that tooling like our planned "failure injection" feature for Petabridge.Cmd can still work.